lib/hooks/useSiteData.ts

```diff
@@ -26,14 +26,23 @@ const useSiteData = (siteUUID: string) => {
     siteListFetcher
   );
 
-  const error = AggregateError([forecastError, siteListError]);
-  const isLoading = isSiteListLoading || isForecastLoading;
+  const {
+    data: clearskyData,
+    error: clearskyError,
+    isLoading: isClearskyLoading,
+  } = useSWR(
+    `${process.env.NEXT_PUBLIC_API_BASE_URL}/api/sites/${siteUUID}/clearsky`,
+    clearskyFetcher
```

Comment:
should be able to delete this line, make `useSWR` use our default fetcher here.

Reply:
Once you remove, make sure to explicitly type the `useSWR` data. Here's an example: `useSWR<ForecastData>(`